expo/gbe/forms.py

```diff
@@ -419,26 +419,37 @@ class VolunteerBidForm(forms.ModelForm):
     error_css_class = 'error'
     title = forms.HiddenInput()
     description = forms.HiddenInput()
-    availability = forms.MultipleChoiceField(
+    available_windows = forms.ModelMultipleChoiceField(
+        queryset=VolunteerWindow.objects.none(), 
         widget=forms.CheckboxSelectMultiple,
-        choices=volunteer_availability_options,
-        label=volunteer_labels['availability'],
-        help_text=volunteer_help_texts['volunteer_availability_options'],
-        required=True)
-    unavailability = forms.MultipleChoiceField(
+        label='Available')
+    unavailable_windows = forms.ModelMultipleChoiceField(
+        queryset=VolunteerWindow.objects.none(),
```

Comment:
help_text=volunteer_help_texts['volunteer_availability_options'],

is gone, so help has disappeared.